scripts/deploy_build

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+set -e
+
+# shellcheck disable=SC1091
+. jenkins_account
+
+# (much of this needs to be extraced into a per-user config file:)
+# for a remote dev cluster:
+#HOST=lotus-31
+#CLUSTER=""
+#CLUSTER="1"
+# empty $HOST for a local Vagrant cluster:
+HOST=${HOST:-""}
+CLUSTER=${CLUSTER:-""}
+CHROMA_MANAGER="${HOST}vm${CLUSTER}3"
+# shellcheck disable=SC2034
+STORAGE_NODES="${HOST}vm${CLUSTER}[5-8]"
+ALL_NODES="${HOST}vm${CLUSTER}[2-9]"
+EE_VERSION=$(make -s -f include/Makefile.version .ieel_version)
+# if you want a release from jenkins:
+#EE_JOB_NUM=292
+# otherwise:
+ARCHIVE_NAME="iml-$EE_VERSION.tar.gz"
+
+CHROMA_DIR=.
+CHROMA_USER="admin"
+CHROMA_EMAIL="brian.murrell@intel.com"
```

Comment:
Can we make this a more generic email?

Reply:
Sure.  Any suggestions of what we have that is more generic?

Reply:
Any suggestions of what to use here @jgrund?

Reply:
sorry, just saw this:

iml@intel.com

Reply:
iml@intel.com is a valid e-mail address?

Reply:
yes.